### ISSUE TITLE
the levelDB unit test is broken

### ISSUE THREAD

**Issue by nan-wang** [jina-ai/jina#535]
**Describe the bug**
The `level` argument is missing in the unittest. 

**Describe how you solve it**

`tests/executors/indexers/keyvalue/test_leveldb.py`

```python
     def test_add_query(self):
-        indexer = LeveldbIndexer(index_filename='leveldb.db')
+        indexer = LeveldbIndexer(index_filename='leveldb.db', level='doc')
         self.run_test(indexer)
```

`/tests/yaml/test-leveldb.yml`

```python
+  level: 'doc'
```

---

**Environment**

**Screenshots**

**Comment by hanxiao** [comment]
related to #610 

please fix it.

**Comment by JoanFM** [comment]
I am fixing it in the scope of #610

**Comment by JoanFM** [comment]
Issue closed in the scope of #610
